ci-operator/step-registry/ipi/install/ipi-install-chain.yaml

```diff
@@ -3,5 +3,6 @@ chain:
   steps:
   - ref: ipi-install-rbac
   - ref: ipi-install-install
+  - ref: ipi-metrics
```

Comment:
Let's think of a more specific name for this. The term "metrics" is too generic.